spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/refresher_spec.rb

```diff
@@ -49,7 +49,7 @@ def assert_specific_vm
     expect(vm.hardware.cpu_total_cores).to eq(2)
     expect(vm.hardware.cpu_sockets).to eq(2)
     expect(vm.hardware.bitness).to eq(64)
-    expect(vm.operating_system[:product_name]).to eq('red-7-amd64')
+    expect(vm.operating_system[:product_name]).to(eq('linux_redhat'))
```

Comment:
This rubocop warning is actually an error, we have spec files excluded because `expect().to eq()` is the right pattern but they aren't being excluded for some reason after our switch to manageiq-style from manageiq/guides.  You can just ignore the rubocop warning about parens for things like this.

Reply:
Ok - so line 67 in the refresher_spec.rb (which was hit by rubocop previously) should also be reverted back to the old style?

```
    expect(vm.labels.count).to(eq(3))
```

Reply:
:+1: yes